src/llmq/utils.cpp

```diff
@@ -151,15 +151,15 @@ std::vector<std::vector<CDeterministicMNCPtr>> ComputeQuorumMembersByQuarterRota
 
     PreviousQuorumQuarters previousQuarters = GetPreviousQuorumQuarterMembers(llmqParams, pBlockHMinusCIndex, pBlockHMinus2CIndex, pBlockHMinus3CIndex, pCycleQuorumBaseBlockIndex->nHeight);
 
-    auto nQuorums = size_t(llmqParams.signingActiveQuorumCount);
-    std::vector<std::vector<CDeterministicMNCPtr>> quorumMembers(nQuorums);
+    size_t nQuorums = static_cast<size_t>(llmqParams.signingActiveQuorumCount);
```

Comment:
I wouldn't do the static_cast here, just need to functional cast `size_t(llmqParams.signingActiveQuorumCount)`

Reply:
IMO functional cast should be used almost never because it is very wide and compiler will almost never show you a mistake by warning or error, you just directly tell compiler "try to do whatever you can do, but I want to convert type A to type B". It's fine for C-code, but in C++ better to avoid C-style casts.

Instead, should be used one of C++ cast:
 - static_cast for integer + enum convertion
 - const_cast if need to remove `const`
 - dynamic_cast for inheritance
 - reinterpret_cast if need to reinterpret data as completely other type. This one is the least strict type of conversion, but it still keep const modifier (and that's a difference with functional/C-style type of cast).

Since C++11 there is one more good way to convert type that control your conversion:
```
size nQuorums = size_t{llmqParams.signingActiveQuorumCount};
```

In this basic example functional cast behave as you expecting:
```
int y = static_cast<int>(x);
int z = int(x);
int z = int{x}; // warning: narrowing conversion of ‘x’ from ‘uint64_t’ {aka ‘long unsigned int’} to ‘int’ [-Wnarrowing]
```
But in other cases, conversion may be broader than you expect to see:
 - pointer to int
```
char *x = nullptr;
size_t y = static_cast<size_t>(x); // error: invalid ‘static_cast’ from type ‘char*’ to type ‘size_t’ {aka ‘long unsigned int’}
size_t z = size_t(x); // WORKS FINE but should not!
size_t z = size_t{x}; // error: invalid conversion from ‘char*’ to ‘size_t’ {aka ‘long unsigned int’} [-fpermissive]
```

 - adding/removing const modifier:
```
const char *x = nullptr;
char *y = static_cast<char*> (x); //  invalid ‘static_cast’ from type ‘const char*’ to type ‘char*’
char *z = const_cast<char*> (x); // OK as expected
int *zz = (int *)x; // OK, but did you expect it?
const int *yy = reinterpret_cast<const int *>(x); // OK as expected
int *yyy = reinterpret_cast<int *>(x); // error: ‘reinterpret_cast’ from type ‘const char*’ to type ‘int*’ casts away qualifiers
using P = char*;
P zzz = P(x); // OK, but is it what expected?
P zzzz = P{x}; // error: invalid conversion from ‘const char*’ to ‘P’ {aka ‘char*’} [-fpermissive]
```

So, I'd prefer to use static/cost/reinterpret_cast whenever possible, and use C-style conversion only in C code.

Reply:
I am 99% sure that this function style cast I talk about here are aliases for static_casts, just shorter and less cumbersome, C in fact doesn't even have these functional casts (such as `int(5)`, but only c-style casts such as `(int)5`). For integer types imo, static_casts are probably overly verbose and don't provide any benefit over function casts